README.md

```diff
@@ -57,6 +57,8 @@ The extension currently offers 4 core features, with more to come.
 - Validation of OpenFGA's `.fga` files
 	- As you type in a `.fga` OpenFGA file in the editor, the extension will highlight errors that make your models invalid 
 
```

Comment:
You can't refer to resources with relative paths. The paths must be absolute so that they are picked up when viewed on the VS Code marketplace and the OpenVSX registry